be/src/connector/hive_connector.h

```diff
@@ -77,6 +77,8 @@ class HiveDataSource final : public DataSource {
     int64_t estimated_mem_usage() const override;
     bool can_estimate_mem_usage() const override { return true; }
 
+    void get_split_tasks(std::vector<pipeline::ScanSplitContextPtr>* split_tasks) override;
```

Comment:
Only for hive connector, who will implement this api for other DataSource, LakeDataSource for example?

Reply:
Anyone who cares can implement it.